packages/plans-grid/src/plans-feature-list/index.tsx

```diff
@@ -19,81 +19,165 @@ import '../types-patch';
  */
 import './style.scss';
 
-const TickIcon = <Icon icon={ check } size={ 17 } />;
-const CrossIcon = <Icon icon={ close } size={ 17 } />;
-const ChevronDown = (
-	<svg width="8" viewBox="0 0 8 4">
-		<path d="M0 0 L8 0 L4 4 L0 0" fill="currentColor" />
-	</svg>
+interface FeatureListItemContentWrapperProps {
+	className: string;
+}
+
+interface FeatureListIconProps {
+	className: string;
+}
+
+const TickIcon: React.FunctionComponent< FeatureListIconProps > = ( { className } ) => (
+	<Icon className={ className } icon={ check } size={ 18 } />
+);
+const CrossIcon: React.FunctionComponent< FeatureListIconProps > = ( { className } ) => (
+	<Icon className={ className } icon={ close } size={ 18 } />
 );
 
-function domainMessageStateMachine(
+const DefaultFeatureListItemContentWrapper: React.FunctionComponent< FeatureListItemContentWrapperProps > = ( {
+	children,
+	className,
+} ) => <span className={ className }>{ children }</span>;
+
+const FeatureAnnualDiscountNudge: React.FunctionComponent< {
+	billingPeriod: Plans.PlanBillingPeriod;
+	__: typeof import('@wordpress/i18n').__;
+} > = ( { billingPeriod, __ } ) => (
+	<span
+		className="plans-feature-list__item-annual-nudge"
+		aria-label={
+			billingPeriod === 'ANNUALLY'
+				? __( 'Included with annual plans', __i18n_text_domain__ )
+				: __( 'Only included with annual plans', __i18n_text_domain__ )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Included with annual plans' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/plans-grid/monthly-prices-try-ETK-workflow-fix)